third_party/2and3/cryptography/hazmat/bindings/openssl/binding.pyi

```diff
@@ -0,0 +1,6 @@
+from typing import Any
+
+class Binding:
+    ffi: Any
+    lib: Any
```

Comment:
You could use `Optional[Any]` here, since `None` is definitely one of the possible values. (Import required.)